app/src/main/java/com/github/factotum_sdp/factotum/ui/picture/PictureFragment.kt

```diff
@@ -15,13 +15,14 @@ import java.io.File
 import java.util.Date
 import java.util.Locale
 
-class PictureFragment : Fragment() {
+class PictureFragment(clientID : String): Fragment() {
```

Comment:
Could clientID be a UUID instead of a String ?

Reply:
For the moment I dont think so, since theDRecord hold the clientID as a string since it's just the client name, but if we change that or if @schuetzcarl add the UUID in what is stored in the DRecord i'm not against this change, but that will me a small an fast change if we do it so for the moment it's not worth bothering

Reply:
Ok, works for me